cnxarchive/sql/search/title.part.sql

```diff
@@ -10,4 +10,4 @@ SELECT
 FROM
   latest_modules
 WHERE
-  name ~* req(%({0})s::text)
+  StripHTML(name) ~* req(%({0})s::text)
```

Comment:
So your saying that we need to flag and incoming query as html and direct the search to that index?  😢 That's kind of a major change. Do you have a plan of attack in mind?  Can't we do a load test first do see if adding this additional function makes a significant difference to search time?